CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ### 2.4.3 (Unreleased)
 
+- [#574](https://github.com/influxdata/clockface/pull/574): Upgrade Storybook to 6.1.17
+
```

Comment:
Range slider updates i think are needed here too

Reply:
This PR isn't to fix the changelog, I just added this entry for the Storybook upgrade. We'll fix all the missing past issues in another PR.